weed/shell/command_volume_server_evacuate.go

```diff
@@ -134,18 +111,34 @@ func (c *commandVolumeServerEvacuate) volumeServerEvacuate(commandEnv *CommandEn
 func (c *commandVolumeServerEvacuate) evacuateNormalVolumes(commandEnv *CommandEnv, volumeServer string, skipNonMoveable, applyChange bool, writer io.Writer) error {
 	// find this volume server
 	volumeServers := collectVolumeServersByDc(c.topologyInfo, "")
-	var thisNodes []*Node
-	thisNodes, c.otherNodes = c.nodesOtherThan(volumeServers, volumeServer)
+	thisNodes, otherNodes := c.nodesOtherThan(volumeServers, volumeServer)
 	if len(thisNodes) == 0 {
 		return fmt.Errorf("%s is not found in this cluster", volumeServer)
 	}
 
 	// move away normal volumes
+	ticker := time.NewTicker(topologyInfoUpdateInterval)
 	for _, thisNode := range thisNodes {
 		for _, diskInfo := range thisNode.info.DiskInfos {
+			if applyChange {
+				select {
+				case <-ticker.C:
```

Comment:
you can use `collectTopologyInfo(commandEnv, 0)` and set a non-zero number for the second parameter.

Reply:
> you can use `collectTopologyInfo(commandEnv, 0)` and set a non-zero number for the second parameter.

I tried like this without checking applyChange
When restarted, it will be updated twice and will have to be run asynchronously so as not to wait for sleep inside collectTopologyInfo

Reply:
> How about waiting a few seconds after each successful move? I prefer not to use the timer since it is not predictable. Some volume may take longer than 5 minutes, some may take 5 seconds. The topology is not always updated.

ok. i remove ticker not important